docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/cpu-device/performance-hint-and-thread-scheduling.rst

```diff
@@ -187,3 +187,18 @@ are executed in parallel.
 
 For details on multi-stream execution check the
 :doc:`optimization guide <../../optimize-inference/optimizing-throughput/advanced_throughput_options>`.
+
+.. _Inference_threads_wait_actively:
+
+Inference threads wait actively
```

Comment:
Since this is generic threading tips page I would ask you to formalize the description in more format way:
1. Case you are describing is an example of serial composability of different threading runtimes. So it should be name of the section.
2. We need to formalize the application that we are focusing on: pipeline with multiple OV inferences interleaved with some other application logic (maybe calls to another library) executed sequentially.
3. We need to describe the reason of performance issues in that scenarion. Actually you already shared some info about active searching for work which takes CPU resources. It worth to explicitly mentioned that it is true for both TBB and OMP so threads migration between areas will happen twice per pipeline iteration.
4. 1ms is very specific to particular library - would avoid detailed numbers
5. Lets describe all possible sotuion:
5.1. Most effective is to use oneTBB for all computations made in pipeline 
5.2. Rebuilding OV with OMP from source is another option,
5.3. Limit number of threads / disable pinning for OV or other parts of the pipeline to let OS do better scheduling.
5.4. In case second runtime is OMP user can set OMP_WAIT_POLICY=PASSIVE to minimize perf gap on OMP->TBB runtime switch.

@wangleis Do you have anything to add?

Reply:
LGTM, Thanks.